File: producto-micro/producto-micro/src/main/java/edu/academik/producto/micro/domain/Product.java
package edu.academik.producto.micro.domain;

import java.io.Serializable;
import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.Id;
import javax.persistence.Table;

/**
 *
 * @author Mario Batres
 */
@Entity
@Table(name = "productos")
public class Product implements Serializable {

    @Id
    @Column(name = "producto_id")
    private Long productId;

    @Column(name = "descripcion")
    private String description;

    public Long getProductId() {
        return productId;
    }

    public void setProductId(Long productId) {
        this.productId = productId;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

}

File: RetrofitClient/app/src/main/java/edu/academik/Product.kt
package edu.academik

import com.fasterxml.jackson.annotation.JsonProperty
import java.io.Serializable

class Product : Serializable {

    @JsonProperty("productoId")
    var productId: Long? = null

    @JsonProperty("codigo")
    var code: String? = null

    @JsonProperty("descripcion")
    var description: String? = null

}